addons/mail/static/src/core/web/thread_model_patch.js

```diff
@@ -11,23 +11,9 @@ patch(Thread.prototype, {
     setup() {
         super.setup();
         this.recipients = Record.many("Follower");
+        this.activities = Record.many("Activity");
```

Comment:
Shouldn't at least one of activities or thread define its inverse to match the other one?

Reply:
and `thread = Record.one("Thread");` is the inverse on the `activity` model.

Reply:
If neither fields have inverse defined on them, they will not be sync together.